examples/example-next-static-export/README.md

```diff
@@ -1,6 +1,6 @@
 # example-next-static-export
 
-This project was generated with `@osdk/create-app` from the `next-static-export` template. It is built against a locally generated SDK and a non-existent Foundry stack so is intended for reference purposes only.
+This project was generated with [`@osdk/create-app`](https://www.npmjs.com/package/@osdk/create-app) from the `next-static-export` template. It is built against a locally generated SDK and a non-existent Foundry stack so is intended for reference purposes only.
```

Comment:
Just for better readability. Same applies for the other examples.

```suggestion
This project was generated with [`@osdk/create-app`](https://www.npmjs.com/package/@osdk/create-app) from the `next-static-export` template. It is built against a locally generated SDK and a non-existent Foundry stack, so it is intended for reference purposes only.
```

Reply:
Done